src/tests/integration/python/test_python.py

```diff
@@ -92,3 +95,48 @@ def test__python_project_install_lint_and_publish(
     kraken_ctx.load_project(directory=tempdir / consumer_dir)
     kraken_ctx.execute([":python.install"])
     # TODO (@NiklasRosenstein): Test importing the consumer project.
+
+
+@unittest.mock.patch.dict(os.environ, {})
+def test__python_project_upgrade_python_version_string(
+    kraken_ctx: Context,
+    tempdir: Path,
+) -> None:
+    project_dir = "version-project"
+    # FIXME - now this is hardcoded also in .kraken.py file, find a way to pass it there from test level
```

Comment:
@NiklasRosenstein any suggestions on this?

Reply:
As an alternative to placing a `.kraken.py` into that folder, you can build up the DAG inside the test. That would allow you to easily specify the build version number in the test.

```suggestion
    # FIXME - now this is hardcoded also in .kraken.py file, find a way to pass it there from test level
    pytest.build(as_version="9.9.9a1")
```

You can use `kraken.core.testing.kraken_project(kraken_ctx, tempdir)` for that and can forego the `kraken_ctx.load_project()` call.